modules/jolt_physics/shapes/jolt_world_boundary_shape_3d.cpp

```diff
@@ -29,22 +29,34 @@
 /**************************************************************************/
 
 #include "jolt_world_boundary_shape_3d.h"
+#include "jolt_custom_material_override_shape.h"
 
 #include "../jolt_project_settings.h"
 #include "../misc/jolt_type_conversions.h"
 
 #include "Jolt/Physics/Collision/Shape/PlaneShape.h"
 
-JPH::ShapeRefC JoltWorldBoundaryShape3D::_build() const {
+void JoltWorldBoundaryShape3D::_set_material(JPH::RefConst<JoltPhysicsMaterial> &material) {
+	if (!jolt_ref) {
+		return;
+	}
+	jolt_ref_mutex.lock();
+	JoltCustomMaterialOverrideShape *ref = static_cast<JoltCustomMaterialOverrideShape *>(jolt_ref.GetPtr());
```

Comment:
Same here (although I'll add the missing accessors for the material to this shape).